src/main/kotlin/solve/importer/view/DirectoryPathView.kt

```diff
@@ -78,17 +89,23 @@ class DirectoryPathView : View() {
                 }
             }
 
+            prefHeight = 133.0
+            prefWidth = 453.0
+            style = "-fx-background-color: 000000"
```

Comment:
Invalid style `WARNING: Caught 'java.lang.ClassCastException: class java.lang.Double cannot be cast to class javafx.scene.paint.Paint (java.lang.Double is in module java.base of loader 'bootstrap'; javafx.scene.paint.Paint is in unnamed module of loader 'app')' while converting value for '-fx-background-color' from inline style on VBox@19808b4
a`